compose/docker-compose.yml

```diff
@@ -19,12 +19,27 @@ services:
       - appdata:/var/www/html
       - sockdata:/sock
       - ssldata:/etc/nginx/certs
+    healthcheck:
+      test: 'curl --fail http://127.0.0.1:8000'
+      interval: 1m30s
+      timeout: 10s
```

Comment:
Same with timeout, I would think these would be consistent between the files unless there was some other reason to do so (which should probably then be documented).

Reply:
Remove all interval, timeout, and start-period, and use default value.

Reply:
👍